### DISCUSSION TITLE
(nexus-repository-migrator) Request for migration

### DISCUSSION THREAD

**Discussion by JPRuskin** [chocolatey-community/chocolatey-packages#2523] [Migrations]
### Why should the package be migrated to this Chocolatey Community Chocolatey Packages Repository?

Nexus Repository is currently broken, due to a lack of migration. After discussion with folk, it was suggested it should be a separate package. The package isn't currently separate, but will be split out.

### What is the link to the package hosted on Chocolatey Community Repository?

nexus-repository-migrator

### What is the link to the package source?

https://github.com/chocolatey-community/chocolatey-packages/pull/2519

### Responsibilities

- [X] I will be responsible for creating a PR to migrate the package.
- [x] I will be available to fix any issues with the package or the automatic updater!

**Comment by AdmiringWorm** [comment]
THis was decided that it should be done.
